src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Syntax/SyntaxSerializer.cs

```diff
@@ -105,7 +105,7 @@ private void WriteNode(SyntaxNode node)
 
     protected virtual void WriteSpan(TextSpan span)
     {
-        WriteValue($"[{span.Start}..{span.End}{Separator}Width: {span.End - span.Start}");
+        WriteValue($"[{span.Start}..{span.End}){Separator}Width: {span.End - span.Start}");
```

Comment:
Did you mean this?
```suggestion
        WriteValue($"[{span.Start}..{span.End}]{Separator}Width: {span.End - span.Start}");
```

Reply:
No, the end is exclusive. I think we use `[x..y)` to demonstrate that in other places as well. Definitely we did that here previously:

https://github.com/dotnet/razor/blob/97c495207d68dc083874a5c1c198378dd9b449be/src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Syntax/SyntaxSerializer.cs#L136

Reply:
Oh right, thanks. Looked weird out of context